python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -564,6 +569,7 @@ class TickData(
             ("run_requests", Optional[Sequence[RunRequest]]),  # run requests created by the tick
             ("auto_materialize_evaluation_id", Optional[int]),
             ("reserved_run_ids", Optional[Sequence[str]]),
+            ("user_canceled", bool),
```

Comment:
maybe should be `user_terminated` or something. just named this `canceled` to get the changes in, but not pleased with the name. i feel like it implies there is a `cancel` operation that can be taken on a tick

Reply:
how about user_interrupted (or just interrupted)?